lib/any_login/providers/authlogic.rb

```diff
@@ -4,14 +4,14 @@ module Authlogic
 
       module Controller
 
-        def self.any_login_current_user_method
-          @@any_login_current_user_method ||= "current_#{AnyLogin.klass.to_s.parameterize.underscore}".to_sym
+        def self.any_login_current_user_method(klass)
+          @@any_login_current_user_method ||= "current_#{klass.to_s.parameterize.underscore}".to_sym
         end
```

Comment:
The updated interface for `any_login_current_user_method` also breaks the custom providers support for anyone using that feature: https://github.com/igorkasyanchuk/any_login#custom-providers

I don't believe it is very heavily used since I recently added it, however could you update the documentation to include the appropriate changes?

Reply:
I updated README.md 🙏 
https://github.com/igorkasyanchuk/any_login/pull/54/commits/0052f2f1a46d251ec73472851d96267c9cb18f1a

Reply:
Thank you! Tested these changes locally and it worked perfect for my custom login provider based application.